packages/manager/modules/gcj/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@ovh-ux/manager-gcj-module",
+  "version": "0.0.1",
+  "private": true,
+  "description": "Generic Customer Journey components and hooks",
+  "homepage": "https://github.com/ovh/manager/blob/master/packages/manager/modules/gcj/README.md",
+  "bugs": {
+    "url": "https://github.com/ovh/manager/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/gcj"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsc",
+    "test": "manager-test run",
+    "test:coverage": "manager-test run --coverage"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-react-components": "^2.27.0"
```

Comment:
MRC can be added in `devDependencies` and `peerDependencies`. instead of a direct deps.